pages/common/aria2c.md

```diff
@@ -0,0 +1,28 @@
+# aria2c
+
+> Fast download utility
+> Supports HTTP(S), FTP, SFTP, BitTorrent, and Metalink
+
+- Download a URL to a file
+
+`aria2c {{url}}`
+
+- Download from multiple sources
+
+`aria2c {{url_1}} {{url_2}}`
+
+- Download Metalink
+
+`aria2c {{magnet:?xt=urn:btih:...}}`
```

Comment:
This, and the previous two could be merged into a single one. After all the magnet link is just another link.

```
- Download to a file from one or more supported URLs
`aria2c {{url_1}} {{url_2}}`
```